cmd/installer/pkg/install/manifest.go

```diff
@@ -188,6 +188,10 @@ func (t *Target) Partition(bd *blockdevice.BlockDevice) (err error) {
 	default:
 		typeID := "AF3DC60F-8384-7247-8E79-3D69D8477DE4"
 		opts = append(opts, partition.WithPartitionType(typeID))
+
+		if t.Size == 0 {
+			opts = append(opts, partition.WithMaximumSize(true))
+		}
```

Comment:
I'm not sure how to best enforce/check, but I was thinking that this option makes sense only if it is the last partition. What happens if a user specifies 2 partitions, and omits `size` for both?

Reply:
probably two sides of that:

1. make sure partitioning code can handle that (return meaningful error that no space is available)
2. config validation to make sure only the last partition can have size of zero

Reply:
Makes sense. We can leave this as-is, and make it a lesser known feature for 0.6. Up to you.

Reply:
added both validation checks